github/repos_comments.go

```diff
@@ -109,6 +116,9 @@ func (s *RepositoriesService) GetComment(owner, repo string, id int) (*Repositor
 		return nil, nil, err
 	}
 
+	// TODO: remove custom Accept header when this API fully launches.
+	req.Header.Set("Accept", mediaTypeReactionsPreview)
+
 	c := new(RepositoryComment)
 	resp, err := s.client.Do(req, c)
 	if err != nil {
```

Comment:
Thanks for the extensive review and testing, Dmitry!  I really appreciate it.

I've added the `ListOptions` and updated the tests, but I don't think any docs need updating based upon the docs found in your example from `activity_events.go`.

According to the docs for `UpdateComment`:
https://developer.github.com/v3/repos/comments/#update-a-commit-comment
it does not return the reactions object.

However, I just ran an experiment... If I add the custom `Accept` header, it *does* return the `Reactions` struct, however it does *not* allow me to modify the `Reactions` during the `UpdateComment`!!!

So I'm thinking it would cause less user confusion if we leave *off* the custom Accept header and let the user get an error if they provide a `Reactions` struct in the provided `RepositoryComment` instead of accepting it and giving them the false hope that they can update it.

Thoughts?

Meanwhile, I will update the code with the latest requested changes.
Thanks again for your reviews!